include/psa/crypto_values.h

```diff
@@ -2021,6 +2021,23 @@
 #define PSA_ALG_TLS12_PSK_TO_MS_GET_HASH(hkdf_alg)                         \
     (PSA_ALG_CATEGORY_HASH | ((hkdf_alg) & PSA_ALG_HASH_MASK))
 
+#define PSA_ALG_TLS12_ECJPAKE_TO_PMS_BASE       ((psa_algorithm_t)0x08000600)
```

Comment:
Do we intend to add this to the official PSA spec?

Reply:
I your question about this particular macro or the new ad-hoc KDF in general?

Reply:
If this is a specialized business requirement of Mbed TLS, it should be called `MBEDTLS_xxx` and be in the vendor range. If this is something many implementations are likely to offer, it should be contributed back to the official PSA spec.

If I understand correctly, this is something every implementation of TLS over ECJPAKE needs, similar to `PSA_ALG_TLS12_PRF` and `PSA_ALG_TLS12_PSK_TO_MS`, and so it should go into the official spec?

Reply:
Yes, it's necessary to implement the ECJPAKE key exchange of TLS 1.2 over the PSA PAKE API.